lib/p2p/Pool.ts

```diff
@@ -875,7 +881,7 @@ class Pool extends EventEmitter {
 
     if (!peer.inbound && peer.nodePubKey && shouldReconnect && (addresses.length || peer.address)) {
```

Comment:
Think we can remove `!peer.inbound` check here since we're doing the same check in `tryConnectNode`.